x/concentrated-liquidity/lp.go

```diff
@@ -322,22 +328,18 @@ func (k Keeper) UpdatePosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 
 	currentTick := pool.GetCurrentTick().Int64()
 
-	// update lower tickInfo state
-	// TODO: come back to sdk.Int vs sdk.Dec state & truncation
```

Comment:
is this for sure no longer something we need to look into?

Reply:
AFAIU, yes, we have conformed to sdk.Int and int64 for the tick states